airbyte-integrations/connectors/source-bigquery/acceptance-test-config.yml

```diff
@@ -5,3 +5,19 @@ tests:
   spec:
     - spec_path: "src/test-integration/resources/expected_spec.json"
       config_path: "src/test-integration/resources/dummy_config.json"
+  connection:
```

Comment:
Could you please follow the latest SAT configuration structure like this file? https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/source_acceptance_test_files/acceptance-test-config.yml.hbs#L1 
It would make it easier to set high test strictness levels afterward. I'm also curious to see if the tests pass in. `high` test strictness level.